aspnetcore/mvc/controllers/actions.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Handle requests with controllers in ASP.NET Core MVC
 author: ardalis
-description: 
+description: Learn how ASP.NET Core MVC controllers and Actions handle requests and return responses.
```

Comment:
@Rick-Anderson Please review and make suggestions to improve this new description.

Reply:
That's pretty good.